src/test/kotlin/me/modmuss50/mpp/test/modrinth/ModrinthTest.kt

```diff
@@ -229,7 +229,7 @@ class ModrinthTest : IntegrationTest {
                     projectId = "12345678"
 
                     minecraftVersionRange {
-                        start = "1.19.4"
+                        start = "1.13.1" // test WALL_OF_SHAME
```

Comment:
This should then test that the correct versions versions were submited bellow. It needs to check the new min version, and now one of the special versions.